apps/system/style/notifications/notifications.css

```diff
@@ -268,7 +268,7 @@
 #notifications-container .notification.disappearing,
 #notifications-container .notification.disappearing-via-clear-all,
 .notification-toaster.disappearing {
-  transition: transform 0.2s linear;
+  transition: transform 0.3s linear;
```

Comment:
Yes `0.2` was pretty short once you only do a fraction of it (with the `durationLeft` stuff)